lib/src/Base/Func/FieldFunctionImplementation.cxx

```diff
@@ -138,14 +141,21 @@ Mesh FieldFunctionImplementation::getInputMesh() const
 /* Accessor for the output mesh */
 void FieldFunctionImplementation::setOutputMesh(const Mesh & outputMesh)
 {
-  throw NotDefinedException(HERE) << "In FieldFunctionImplementation::setOutputMesh";
+  if (outputMesh.getDimension() != outputMesh_.getDimension())
+    throw InvalidArgumentException(HERE) << "Mesh must have the same dimension";
+  outputMesh_ = outputMesh;
 }
 
 Mesh FieldFunctionImplementation::getOutputMesh() const
 {
   return outputMesh_;
 }
 
+Bool FieldFunctionImplementation::isPointWise() const
+{
```

Comment:
why "w" is in capital letters ? isPointwise or isActingPointwise?

Reply:
I agree, isActingPointwise is better